Tests/LegacyUnit/Service/RegistrationManagerTest.php

```diff
@@ -6353,16 +6354,15 @@ public function createRegistrationSavesRegistration()
             $subject->getRegistration()
         );
         $uid = $subject->getRegistration()->getUid();
-        self::assertTrue(
+        /** @var ConnectionPool $connectionPool */
+        $connectionPool = GeneralUtility::makeInstance(ConnectionPool::class);
+        $connection = $connectionPool->getConnectionForTable('tx_seminars_attendances');
+        $result = $connection->count('*', 'tx_seminars_attendances', ['uid' => $uid]);
         // We're not using the testing framework here because the record
         // is not marked as dummy record.
```

Comment:
This comment needs to be above the line with the `count`, not below, as is is referring to that line.

Reply:
This comment still needs to be moved.